packages/cordova-plugin-aerogear-metrics/www/metrics.js

```diff
@@ -0,0 +1,42 @@
+
+/**
+ * Get device metrics, to be called after deviceReady event
+ * 
+ * It uses cordova-plugin-device plugin.
+ * @example
+ * const deviceMetrics = await aerogear.getDeviceMetrics();
+ * @returns {Promise<DeviceMetrics>} The device metrics
+ */
+exports.getDeviceMetrics = function () {
+  return new Promise(function (res, rej) {
+    res({
+      platform: device.platform,
+      platformVersion: device.version,
+      device: device.model
+    });
+  });
```

Comment:
`return Promise.resolve({...})` for a shortcut.

Reply:
Good one!